packages/backend-app-api/src/wiring/BackendInitializer.ts

```diff
@@ -165,4 +167,24 @@ export class BackendInitializer {
 
     return orderedRegisterInits;
   }
+
+  async stop(): Promise<void> {
+    if (!this.#started) {
+      return;
+    }
+    this.#started = false;
```

Comment:
Overly cautious perhaps but could even refuse to re-`start` after. Like a `state: 'idle' | 'started' | 'stopped'`. Or how about thoroughly `state: 'idle' | 'starting' | 'started' | 'stopping' | 'stopped'` capturing the async nature of the operations

Reply:
Switched to not allowing re-starts at all.